modules/installation-azure-config-yaml.adoc

```diff
@@ -175,93 +224,137 @@ endif::openshift-origin[]
+====
+endif::azure-restricted[]
+ifndef::azure-restricted[]
 <4> Whether to enable or disable simultaneous multithreading, or `hyperthreading`. By default, simultaneous multithreading is enabled to increase the performance of your machines' cores. You can disable it by setting the parameter value to `Disabled`. If you disable simultaneous multithreading in some cluster machines, you must disable it in all cluster machines.
 +
 [IMPORTANT]
 ====
 If you disable simultaneous multithreading, ensure that your capacity planning accounts for the dramatically decreased machine performance. Use larger virtual machine types, such as `Standard_D8s_v3`, for your machines if you disable simultaneous multithreading.
 ====
+endif::azure-restricted[]
+ifdef::azure-restricted[]
+<4> You can specify the size of the disk to use in GB. Minimum recommendation for control plane nodes is 1024 GB.
+endif::azure-restricted[]
+ifndef::azure-restricted[]
 <5> You can specify the size of the disk to use in GB. Minimum recommendation for control plane nodes is 1024 GB.
+endif::azure-restricted[]
+ifdef::azure-restricted[]
+<5> The configuration for the machines that comprise compute. The `compute` section is a sequence of mappings. To meet the requirements of the different data structures, the first line of the `compute` section must begin with a hyphen, `-`. If you do not provide these parameters and values, the installation program provides the default value.
+endif::azure-restricted[]
 //To configure faster storage for etcd, especially for larger clusters, set the
 //storage type as `io1` and set `iops` to `2000`.
 <6> Specify a list of zones to deploy your machines to. For high availability, specify at least two zones.
 <7> The cluster network plugin to install. The supported values are `OVNKubernetes` and `OpenShiftSDN`. The default value is `OVNKubernetes`.
 <8> Specify the name of the resource group that contains the DNS zone for your base domain.
 <9> Specify the name of an already existing resource group to install your cluster to. If undefined, a new resource group is created for the cluster.
-ifdef::vnet,private,gov[]
+ifdef::vnet,private,gov,azure-restricted[]
 <10> If you use an existing VNet, specify the name of the resource group that contains it.
 <11> If you use an existing VNet, specify its name.
 <12> If you use an existing VNet, specify the name of the subnet to host the control plane machines.
 <13> If you use an existing VNet, specify the name of the subnet to host the compute machines.
-endif::vnet,private,gov[]
-ifdef::private,gov[]
+endif::vnet,private,gov,azure-restricted[]
+ifdef::private,gov,azure-restricted[]
 <14> You can customize your own outbound routing. Configuring user-defined routing prevents exposing external endpoints in your cluster. User-defined routing for egress requires deploying your cluster to an existing VNet.
```

Comment:
The content for restricted network will be different:

`When using Azure Firewall to restrict Internet access, outbound routing is needed to be configured to be sent via the Firewall. Configuring user-defined routing prevents exposing external endpoints in your cluster. User-defined routing for egress requires deploying your cluster to an existing VNet.`